js/templates/mobile-nav.hbs

```diff
@@ -78,31 +78,20 @@
 
 <div id="nav-advanced-receipts" class="site-nav__panel site-nav__panel--sub" aria-hidden="true">
   <button class="site-nav__back button--back button--alt-primary js-panel-close" aria-controls="nav-advanced-receipts">Advanced data</button>
-  <h3 class="site-nav__title"><a class="site-nav__link--underline" href="{{webAppUrl}}/receipts/?is_individual=true&max_date={{today}}">Receipts - all</a></h3>
+  <h3 class="site-nav__title"><a class="site-nav__link--underline" href="{{webAppUrl}}/receipts/?max_date={{today}}">Receipts - all</a></h3>
```

Comment:
Lets also remove `?max_date={{today}}` since that will be set by the page automatically.